deploy/modo-server/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ghcr.io/sdsc-ordes/smoc-api:latest
+
+WORKDIR /app
+COPY ./server.py /app
+
+USER root
+
+RUN pip install \
```

Comment:
Good point! We also need the ability to verify that the client and server modo images are using the same modo api version.

Reply:
I used the "compatible release operator" ([PEP440](https://peps.python.org/pep-0440/#compatible-release)) to pin server dependencies.

This way, we should benefit from security patches and bugfixes (minor version bumps) without allowing breaking changes (major version bumps).

> Note: s3fs uses [CalVer](https://calver.org/) instead of SemVer. AFAIK, we cannot tell which versions are introducing breaking changes  so I just pinned to the exact version...

Reply:
Perfect! Lokks great now